searchcore/src/vespa/searchcore/proton/server/storeonlyfeedview.cpp

```diff
@@ -585,19 +605,20 @@ StoreOnlyFeedView::internalRemove(FeedToken token, const RemoveOperationWithDocI
          rmOp.getSubDbId(), rmOp.getLid(), rmOp.getPrevSubDbId(), rmOp.getPrevLid(), _params._subDbId);
 
     PendingNotifyRemoveDone pendingNotifyRemoveDone = adjustMetaStore(rmOp, docId.getGlobalId(), docId);
+    auto uncommitted = _pendingLidsForCommit->produce(rmOp.getLid());
```

Comment:
This is named commitToken other places in this file. Any reason we use another name here? If not, consider consolidating.

Reply:
Note, there are more places we use _uncomitted_ in this file.

Reply:
Unified on uncommitted.